torch/csrc/Device.cpp

```diff
@@ -267,6 +274,7 @@ void THPDevice_init(PyObject* module) {
     throw python_error();
   }
   Py_INCREF(&THPDeviceType);
+  THPUpperModuleOfDevice = PyImport_ImportModule("torch");
```

Comment:
You don't have to do this, just stash module somewhere durable and use that

Reply:
Thanks for the review.
Declaration and assignment have merged.

Reply:
OK, I have made the change